### DISCUSSION TITLE
Incus support

### DISCUSSION THREAD

**Discussion by maxlivi** [lxdware/lxd-dashboard#83] [General]
Hello,

First thank you for lxd-dashboard!

Do you plan to support incus now that LXD is no longer part of the Linux Containers project ?

[https://discuss.linuxcontainers.org/t/lxd-is-no-longer-part-of-the-linux-containers-project/17593/9](url)

I have already migrated to incus and on  lxd-dashboard I can see the storage, network, images, profiles but not the VMs and the CTs.

Cheers

Max

**Comment by ingobaab** [comment]
Maybe the wrong place, but: What is technical the difference (already) between the fresh fork?! Really, does it already differ from LXD/LXC? I am just curious

**Reply by maxlivi** [reply]
Hello,

well... there are already many differences, they removed some Canonical stuff like RBAC, CANDIS, MASS, etc etc.

lxc is now incus and lxd incusd.

They switched from dqlite to cowsql.

Many other changes can be seen here:

[https://github.com/lxc/incus/issues?page=4&q=](url)

Cheers

**Comment by matthewalanpenning** [comment]
maxlivi,
Yes, I do plan to support incus as well, but have not compared all the differences between the two APIs yet. From what I read from the incus team they were going to make some changes to correct some design flaws from early LXD development so I was waiting until they get incus closer to a stable version before I added those changes.

**Reply by maxlivi** [reply]
Thanks Mattew
